--- origsrc/xsp-1.9.1/configure.in	2008-04-04 12:26:00.000000000 -0500
+++ src/xsp-1.9.1/configure.in	2008-04-22 23:37:43.171875000 -0500
@@ -61,7 +61,7 @@
 fi
 
 case "$host" in
-	*-*-mingw*|*-*-cygwin*)
+	*-*-mingw*)
 	platform_win32=yes	
 	;;
 esac
--- origsrc/xsp-1.9.1/unittests/Tests.XSP.Security/Makefile.am	2007-11-16 17:11:20.000000000 -0600
+++ src/xsp-1.9.1/unittests/Tests.XSP.Security/Makefile.am	2008-04-22 23:37:43.171875000 -0500
@@ -9,7 +9,7 @@
 unittests_DATA = $(security_test_sources) Tests.XSP.Security.dll Makefile
 
 Tests.XSP.Security.dll: $(security_test_sources)
-	mcs -debug -pkg:mono-nunit -out:$@ -t:library $(security_test_sources)
+	mcs -debug -pkg:mono-nunit -out:$@ -t:library $^
 
 run-test: Tests.XSP.Security.dll
 	nunit-console Tests.XSP.Security.dll
